generic/sysbench.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2017 IBM
+# Author: Hariharan T.S.  <harihare@in.ibm.com>
+
+
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Sysbench(Test):
+
+    """
+    sysbench supports following performance tests
+    :cpu
+    :threads
+    :oltp
+    :fileio
+
+    :avocado: tags=cpu,threads
+    """
+
+    def clear_dmesg(self):
+        process.run("dmesg -c ", sudo=True)
+
+    def verify_dmesg(self):
+        self.whiteboard = process.system_output("dmesg")
+        pattern = ['WARNING: CPU:', 'Oops',
+                   'Segfault', 'soft lockup', 'Unable to handle']
+        for x in pattern:
```

Comment:
can this variable has valid name as `X`  is not looks good ? it can be `fail_pattern`

Reply:
Fixed as per review commit.